design/batch_retrieval_design.md

```diff
@@ -288,7 +288,31 @@ The following integration tests will be implemented in Secrets Provider for K8s
 
 ## Performance tests
 
---
+The following performance tests will be run using *Jmeter*.
+These tests will be run for both the new API and the existing one, and compare the results between them.
+
+#### Benchmark
+
+| Section     | Parameter                      | Value          |
+| ----------- | ------------------------------ | -------------- |
+| Conjur      | \# of standbys                 | 2              |
+| Conjur      | \# of followers                | 1              |
+| Environment | Kubernetes                     | OpenShift 3.11 |
+| Environment | \# of cores                    | 16             |
+| Environment | \# of nodes                    | 1              |
+| Test        | \# of requesting threads       | 20             |
+| Test        | \# of variables                | 1025           |
+| Test        | \# of variabels per policy     | 5              |
+| Test        | \# of policies                 | 205            |
+| Test        | Variable ID length             | 50             |
+| Test        | Variable value length (secret) | 50             |
+
```

Comment:
@abrahamko do you plan to add a benchmark chart for the Secrets Provider performance tests? I am filling the SP performance test flows. For SP will we also be using Jmeter? How many SP will be deployed?

Reply:
For SP, we should run the same performance tests that were defined for SLA.

Reply:
this should be stated in the file either copy and pasted or if the file exists provide the link to the file